runtime/coverage.go

```diff
@@ -103,6 +103,8 @@ func NewLocationCoverage(lineHits map[int]int) *LocationCoverage {
 	}
 }
 
+type LocationInspectionHandler func(location Location) bool
```

Comment:
I wasn't sure what the purpose of the "handler" was, until I read the description of how it is used in the comment below https://github.com/onflow/cadence/pull/2474/files#diff-dd50d117359d58b9ec20ba8e56dc4b6b224a84ca819d943436617baec41d34d5R165. 

Maybe "filter" is a more specific name here? Maybe rename to just `LocationFilter`, and rename the `handler` field below to `LocationFilter` as well, so it is exported, like the other fields.

Reply:
Updated in https://github.com/onflow/cadence/pull/2474/commits/1a19dc8c17180fe902474cf87003ff64e5088989